src/ansi-c/library/stdio.c

```diff
@@ -379,20 +391,27 @@ int __VERIFIER_nondet_int();
 
 inline int fileno(FILE *stream)
 {
-  // just return nondet
-  __CPROVER_HIDE:;
+// just return nondet
+__CPROVER_HIDE:;
+  if(stream == stdin)
+    return 0;
+  else if(stream == stdout)
+    return 1;
+  else if(stream == stderr)
+    return 2;
+
```

Comment:
As long as stdin etc. are uninitialised, the above returns may all be executed.

Reply:
I'm working on a follow-up PR to initialise them.